### PR TITLE
Added Nitesh Thapliyal github profile

### PR DESCRIPTION
**Pull request by Nitesh-thapliyal**
Issue number #205 

Added Github profile

### CODE REVIEW

**Comment by AllanRegush** [comment]
Thank you for the PR 👍 But could you move your entry to the end? Thanks 😄

**Comment by Nitesh-thapliyal** [comment]
Ok done👍

File: data.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "imageUrl": "https://user-images.githubusercontent.com/624760/88123456-d40df580-cbc2-11ea-9add-a7fc8675b243.png",
     "issueNumber": 12
   },
+
```
Comment (AllanRegush) [inline data.json:14]: I don't think we need this blank line.

**Review by AllanRegush — COMMENTED** [review]
Looking Good 👍

**Comment by AllanRegush** [comment]
Looks like there is a merge conflict. Could you resolve this? If you need any help let us know 😄

**Comment by Nitesh-thapliyal** [comment]
Resolved merge conflict🙌

**Comment by eddiejaoude** [comment]
Thank you both 👍

**Pull request merged**